packages/http-event-normalizer/index.js

```diff
@@ -21,6 +21,7 @@ const httpEventNormalizerMiddleware = () => {
     // event.headers ??= {} // Will always have at least one header
```

Comment:
@phawxby Did you see this comment?

I was asking `How are you triggering a lambda without headers in an HTTP event?` to better understand how you're able to create an HTTP event without any headers. As far as I'm aware, it's impossible. I'm really curious how you're able to trigger lambda through APIG/ALB/Function URL without headers.

Others have run into something similar which turned out be their unit tests being incomplete.

Reply:
Ah, that make so much sense now. Middlewares with `http-*` were built to be used with APIG/ALB/Function URL and the event they create. These middlewares are overkill for a lambda that gets called directly with Invoke.

I'm going to close this PR for now then.